api/package.json

```diff
@@ -201,6 +202,7 @@
 		"@types/uuid-validate": "0.0.1",
 		"@types/wellknown": "0.5.4",
 		"@vitest/coverage-c8": "0.25.2",
+		"content-disposition": "0.5.4",
```

Comment:
Can we achieve this without introducing two new dependencies?

Reply:
Very good point! Avoiding new dependencies is a concern for me as well! I'll happy to take a second look at this 👍

Reply:
It does quite a bit more than just format the string, and seeing it's MIT and already a dep of express anyways, it should be good to go 👍🏻